app/form_models/school_search_form.rb

```diff
@@ -42,7 +42,7 @@ def key_stage_options
   end
 
   def job_availability_options
-    [true, false].map { |i| [i.to_s, I18n.t(i, scope: "organisations.filters.job_availability.options")] }
+    [true].map { |i| [i.to_s, I18n.t(i, scope: "organisations.filters.job_availability.options")] }
```

Comment:
this now looks strange - could we not remove this method and replace its usage with I18t("organisations.filters.job_availability.options.true") ?

Reply:
:recycle: